video_creation/final_video.py

```diff
@@ -124,14 +127,45 @@ def make_final_video(
     final = CompositeVideoClip([background_clip, image_concat])
     title = re.sub(r"[^\w\s-]", "", reddit_obj["thread_title"])
     idx = re.sub(r"[^\w\s-]", "", reddit_obj["thread_id"])
+    title_thumb = reddit_obj["thread_title"]
 
-    filename = f"{name_normalize(title)[:251]}.mp4"
+    filename = f"{name_normalize(title)[:150]}"
```

Comment:
```suggestion
    filename = f"{name_normalize(title)[:251]}"
```
On suggestion of Jason, 251 as a limit was specifically chosen.